src/PSR7/Exception/InvalidServerRequestMessage.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace League\OpenAPIValidation\PSR7\Exception;
+
+use Exception;
+use Throwable;
+
+class InvalidServerRequestMessage extends Exception
+{
+    public static function fromOriginal(Throwable $e) : self
```

Comment:
I'm not sure whether `Throwable` is not overkill there. Since we do a rethrow, previous `ValidationFailed` would cover all the cases and will be consised with the message - not any throwable is failed validation

Reply:
Yeah, true. The current way completely hides ANY errors possible: some runtime problems, actual validation problems, syntax errors, god knows what else can go wrong in there. I don't mind if we change this and catch `ValidationError` only.